jaq-json/tests/funs.rs

```diff
@@ -77,3 +77,46 @@ fn tojson() {
     give(json!(0), "0.0 / 0.0 | tojson", json!("null"));
     give(json!(0), "1.0 / 0.0 | tojson", json!("null"));
 }
+
+yields!(
+    math_rem,
+    "def f: -2, -1, 0, 2.1, 3, 4000000001; [f as $a | f as $b | try ($a % $b) catch .]",
+    json!([
+        0,
+        0,
+        "cannot calculate -2 % 0",
+        -2.0,
+        -2,
+        -2,
+        -1,
+        0,
+        "cannot calculate -1 % 0",
+        -1.0,
+        -1,
+        -1,
+        0,
+        0,
+        "cannot calculate 0 % 0",
+        0.0,
+        0,
+        0,
+        0.10000000000000009,
+        0.10000000000000009,
+        "cannot calculate 2.1 % 0",
+        0.0,
+        2.1,
+        2.1,
+        1,
+        0,
+        "cannot calculate 3 % 0",
+        0.8999999999999999,
```

Comment:
Bad idea portabilitywise to end up with these kinds of values in tests?

Reply:
Yes. One trick to make this portable is to use something like `. * 1000 | round`, potentially followed by `. / 1000`.

Reply:
👍 switch to using that